styles/styles.css

```
@@ -0,0 +1,610 @@
+    background-color: #FFFFFF;
+    color: #000000;
+}
+
+.navigation-list{
+    display: flex;
+    margin-left: 70px;
+    margin-right: 70px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    max-width: 450px;
+    justify-content: space-evenly;
+}
+
+.navigation-user {
+    display: flex;
+    margin-right: 70px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    max-width: 350px;
+    margin-left: auto;
+    justify-content: space-evenly;
+}
+
+.navigation-link {
+    font-family: "PT Sans", sans-serif;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000000;
+    text-decoration: none;
+}
+
+
+.navigation-search {
+    background: url(../images/search.svg) no-repeat;
+    width: 18px;
+}
+
+.navigation-likes {
+    background: url(../images/love.svg) no-repeat;
+    width: 18px;
+}
+
+
+
+.navigation-choice {
+    font-family: inherit;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 8px 20px;
+    width: 160px;
+    text-decoration: none;
+    background-color: #756157;
+    color: #FFFFFF;
+    text-transform: uppercase;
+}
+
+.reservations-button {
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 36px;
+    text-align: center;
+    width: 376px;
+    padding: 8px 50px;
+    background-color: #756157;
+    color:#FFFFFF;
+    text-decoration: none;
+}
+
+.about-hotel {
+    text-decoration: none;
+    text-align: center;
+    flex-direction: row;
+    padding: 8px 50px;
+    width: 140px;
+    background-color: #756157;
+    color: #FFFFFF;
+}
+
+.add-to-favourites,
+.added-to-favourites {
+    text-decoration: none;
+    text-align: center;
+    padding: 8px 50px;
+    width: 140px;
+    background-color: #82B3D3;
+    color: #FFFFFF;
+}
+
+.added-to-favourites {
+    background-color: #7DB54F;
+
+}
+
+.button {
+    font-family: inherit;
+    padding: 8px 50px;
+    background-color: #82B3D3;
+    color:#FFFFFF;
+    font-size: 20px;
+}
+
+.main-index {
+    background-color: #FFFFFF;
+}
+
+.welcome {
+    background-image: url(../images/index-background.jpg);
+    background-repeat: repeat;
+    background-size: cover;
+    background-position: top;
+}
+
+.welcome-text {
+    display: block;
+    margin-left: auto;
+    margin-right: auto
+}
+
+
+
+.advantages-service-list {
+    display: flex;
+}
+
+
+.advantages-item-first,
+.advantages-item-fifth {
+    background-color: #82B3D3;
+    color:#FFFFFF;
+}
+
+.advantages-item-second,
+.advantages-item-fourth {
+    background-color: rgba(131, 179, 211, 0.12);
+}
+
+.advantages-item-third {
+    background-color: rgba(131, 179, 211, 0.2);
+}
+
+.advantages-service-item {
+    width: 400px;
+}
+
+.advantages-service-item-first,
+.advantages-service-item-third {
+    background-color: rgba(131, 179, 211, 0.12);
+    color:#FFFFFF;
+}
+
+.advantages-service-item-first {
+    background-image: url(../images/house.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
```

```suggestion
.advantages-service-item-first {
    background-image: url(../images/house.svg);
    background-repeat: no-repeat;
    background-size: ???;
}
```

Разве cover?
И названия классов тут лучше задать по смыслу. 
Ведь иконка домика у этого пункта не потому что он первый, а потому что он про жильё. 
И у аналогичных.